img2dataset/main.py

```diff
@@ -86,7 +86,7 @@ def download(
     incremental_mode: str = "incremental",
     max_shard_retry: int = 1,
     user_agent_token: Optional[str] = None,
-    disallowed_header_directives: Optional[List[str]] = None,
+    disallowed_header_directives: List[str] = ["noai", "noimageai", "noindex", "noimageindex"],
```

Comment:
Using a list as a default value is an anti pattern in python, put back None here
and instead do 
```
if disallowed_header_directives is None:
  disallowed_header_directives = ["noai", "noimageai", "noindex", "noimageindex"]
```
below

Reply:
Thanks for the tip 👍